src/controllers/player.coffee

```diff
@@ -481,7 +489,7 @@ Matching.controller 'matchingPlayerCtrl', ['$scope', '$timeout', '$sce', ($scope
 			Materia.Score.submitQuestionForScoring(qsetItems[i].id, mappedQsetItemText, mappedQsetAudioString)
 		Materia.Engine.end true
 
-	_shuffle = (qsetItems) ->
+	$scope._shuffle = (qsetItems) ->
```

Comment:
I'm not _crazy_ about the shuffle function being attached to scope, since it appears that was only done to facilitate testing. My first reaction is the manner in which you're conducting the test should change so you don't have to directly reference the function (ex: compare the initial qset coming from `qset.data` with the qset that's provided after the `start` method is called).

However, if shuffle has to remain attached to scope, you should drop the prepended underscore. We usually define functions as `_function` to indicate private methods: ones that are only referenced internally. `$scope._shuffle` should just be `$scope.shuffle`. You'll have to update every other reference accordingly.

Reply:
Good point. I opted for reverting the naming for the function, and deleted the previous test function since it wasn't going to work without `_shuffle` being public (not as is anyways). I believe I can't go with the first idea since it's possible the updated array of questions/answers has the exact same order as the initial one, so I need some way to tell if the function is called. 

Coverage is at passing level without code to test that function. That being said, it seems like there's a way to pull in this function while keeping it private without attaching it to scope, but I wanted to bring that up to hear your input first.